docs/docs/References/Plugin_Config.md

```diff
@@ -20,6 +20,7 @@ The name of the file can be specified during the build. By default, the build lo
 |language|Y|enum|Must be `PYTHON27`.|
 |defaultLocale|N|enum|The locale to be used by the plugin if the Delphix user does not specify one. Plugin messages will be displayed in this locale by default. The default value is `en-us`.|
 |rootSquashEnabled|N|boolean|This dictates whether "root squash" is enabled on NFS mounts for the plugin (i.e. whether the `root` user on remote hosts has access to the NFS mounts). Setting this to `false` allows processes usually run as `root`, like Docker daemons, access to the NFS mounts. The default value is `true`. This field only applies to Unix hosts.|
+|extendedStartStopHooks|N|boolean|This controls whether the user's pre-start and post-start hooks will run during enable operations (and, likewise, whether pre-stop and post-stop hooks will run during disable operations). The default value is `false`.|
```

Comment:
Do you think it would make sense to reference the Delphix Engine documentation about hooks: https://docs.delphix.com/docs/datasets/getting-started/hook-scripts-for-automation-and-customization?

Reply:
I'd be nervous that the hooks docs would move in the future, and leave us with a broken link.  It looks like we currently are only linking plugin developers to the top-level `docs.delphix.com` page, and never linking inside of that.

(Actually, there is one place we are linking inside of it... we are telling  SDKdevelopers that they can get `git` set up by following the instructions in the engineering handbook... but I don't think the engineering handbook is available except to Delphix engineers. So, I think this is actually a mistake.)